### DISCUSSION TITLE
Ellipticity in SEP

### DISCUSSION THREAD

**Discussion by fedeaero** [sep-developers/sep#125] [Q&A]
Hi, 

I was wondering if there is an ellipticity field to add to SEP routine, since I only want to detect the streak of the object passing through the field of view, and discard the stars. Therefore I need an ellipticity < 0.3.

Thank you very much!

**Comment by kbarbary** [comment]
The `a` and `b` parameters returned by [`sep.extract()`](https://sep.readthedocs.io/en/v1.0.x/api/sep.extract.html#sep-extract) are proportional to the semi-major and semi-minor axes, so ellipticity can be calculated as:
```python
ellipticity = (objects["a"] - objects["b"]) / objects["a"]
```
